keyboards/mechmini/v2/keymaps/lbibass_625_space/config.h

```diff
@@ -0,0 +1,6 @@
+#include "../../config.h"
```

Comment:
You don't need this include, because of how the compiler handles things. But you do want an include guard here:
```suggestion
#pragma once
```